src/components/__tests__/__snapshots__/Text.test.tsx.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Text Renders Correctly 1`] = `
+<Text
+  style={
+    Array [
+      Object {
+        "display": "flex",
+      },
+    ]
+  }
+/>
+`;
```

Comment:
Interesting! Hopefully not to tedious to keep up to date.

Reply:
It's automated ;)